FFXIV-TRUE-BLUE/class-true-blue.xml

```diff
@@ -578,4 +578,110 @@
+        <supports>True Blue Mage Archetype</supports>
+        <description>
+            <p>While most blue mages enjoy the rush of facing their foes up close and personal, the lore master knows a better way. Through careful observation, reading, and analysis it is possible to gain an unprecedented understanding of the inner beast with none of the undignified quarreling. While they might not learn lore as quickly as others, they are able to draw extra benefits from the lore discover.</p>
+            <div element="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_ARCANE_AZURE" />
+            <div element="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_LORE_MASTER_LEARNING_STYLE" />
+            <div element="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_MONSTROUS_CUNNING" />
+            <div element="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_SPELLSENSE" />
+            <div element="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_ESSENCE_FIELD" />
+            <div element="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_EXTRAPOLATE_LORE" />
+            <div element="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_KNOWLEDGE_IS_POWER" />
+        </description>
+        <sheet display="false" />
+        <rules>
+            <grant type="Archetype Feature" id="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_ARCANE_AZURE" level="1" />
+            <grant type="Archetype Feature" id="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_LORE_MASTER_LEARNING_STYLE" level="1" />
+            <grant type="Archetype Feature" id="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_MONSTROUS_CUNNING" level="1" />
+            <grant type="Archetype Feature" id="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_SPELLSENSE" level="6" />
+            <grant type="Archetype Feature" id="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_ESSENCE_FIELD" level="10" />
+            <grant type="Archetype Feature" id="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_EXTRAPOLATE_LORE" level="14" />
+            <grant type="Archetype Feature" id="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_KNOWLEDGE_IS_POWER" level="20" />
+        </rules>
+    </element>
+    <element name="Arcane Azure" type="Archetype Feature" source="The Blue Mage: True Blue Update" id="ID_FFXIV_ARCHETYPE_FEATURE_TRUE_BLUE_MAGE_ARCANE_AZURE">
+        <description>
+            <p>When you choose this discipline at 1st level, you learn two cantrips from the wizard spell list, your spellcasting ability for these cantrips is Intelligence.</p>
+        </description>
+        <sheet>
+            <description>You learn two cantrips from the wizard spell list, your spellcasting ability for these cantrips is Intelligence.</description>
+        </sheet>
+        <rules>
+            <select type="Spell" name="Cantrip (Arcane Azure)" supports="$(spellcasting:list), 0" level="1" number="2" spellcasting="Wizard" />
+        </rules>
+    </element>
```

Comment:
Did you test if Arcane Azure works properly? 
For the spell "Azure Minora" i had to add: 
        <spellcasting name="Wizard" ability="Intelligence" allowReplace="true">
			<list>Wizard</list>
	</spellcasting>
But this could have been the case as it is a spell and not an Archetype Feature

Reply:
Yeah so I was wrong. What can be done instead is actually what 1/3 casters do when grabbing from a spell list. In this case of Eldritch Knight, I copied some code to make it look like: 
```
<rules>         
     <select type="Spell" name="Cantrip (Arcane Azure)" supports="(Wizard||$(spellcasting:list)), 0" level="1" 
     number="2" spellcasting="True Blue Mage" />
</rules>
```
I also made changes to this spell so hopefully it works as intended.